src/icons.js

```diff
@@ -32,12 +32,12 @@ export function getNodeIcon(hierarchy) {
         'cloudify.nodes.Volume': '\ue620',
         'cloudify.nodes.WebServer': '\ue622',
         'cloudify.nodes.Component': '\ue631',
-        'cloudify.nodes.SharedResource': '\ue632'
+        'cloudify.nodes.SharedResource': '\ue632',
     };
     const defaultNodeType = 'cloudify.nodes.ApplicationModule';
 
     const typeHierarchy = Array.isArray(hierarchy) ? hierarchy : [hierarchy || ''];
-    const knownType = typeHierarchy.find(type => !!nodeTypeToChar[type]);
+    const knownType = typeHierarchy.find((type) => !!nodeTypeToChar[type]);
```

Comment:
Just wondering - why such changes?

Reply:
In Prettier 2.x some rules have been changed. In this case it was: [Change default value for arrowParens to always](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev). I think in our case it would be good to revert to previous behavior (as we just introduced a lot of changes due to ESLint issues fixes).

I took that opportunity to introduce common prettier configuration. I will update all the projects within this issue as it is strictly connected with ESLint in our configuration.

Reply:
What about the trailing comma after last object property (line 35)? It seems it has also changed.

Reply:
Yep, added as well.